app/src/main/res/layout/card_layout.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Again, use `ConstraintLayout` as root layout. And, btw `RelativeLayout` is legacy now